packages/probing-frontend/src/pages/MonitoringDetailPage/components/MonitoringEserviceProbing.tsx

```diff
@@ -0,0 +1,118 @@
+import type { ProbingEservice } from '@/api/monitoring/monitoring.models'
+import { Alert, Box, Chip, Grid, Typography } from '@mui/material'
+import { ButtonNaked } from '@pagopa/mui-italia'
+import { useTranslation } from 'react-i18next'
+import RefreshIcon from '@mui/icons-material/Refresh'
+import { formatDateString } from '@/utils/date.utils'
+import type { TranslationKeys } from './MonitoringInformationContainer'
+import { MonitoringInformationContainer } from './MonitoringInformationContainer'
+
+export const MonitoringEserviceProbing = ({
+  eservicesProbingDetail,
+  refetch,
+}: {
+  eservicesProbingDetail: ProbingEservice | undefined
+  refetch: () => Promise<void>
+}) => {
+  const { t } = useTranslation('common', {
+    keyPrefix: 'detailsPage',
+  })
+
+  const getChipColor = (value: string) => {
+    switch (value) {
+      case 'ONLINE':
+      case 'true':
+        return 'success'
+      case 'OFFLINE':
+      case 'false':
+        return 'error'
+      case 'N/D':
+        return 'warning'
+    }
+  }
+
+  const getChipLabel = (value: boolean) => {
+    switch (value) {
+      case true:
+        return 'active'
+      case false:
+        return 'suspended'
+    }
+  }
```

Comment:
Since the boolean will always have 2 cases, using a switch here may be a bit too much.
I would remove this function and use a ternary

`label={t(eservicesProbingDetail?.probingEnabled ? "active" : "suspended")}`